Button/Button.js

```diff
@@ -121,13 +122,17 @@ const ButtonBase = kind({
 	},
 
 	computed: {
-		className: ({backgroundOpacity, highlighted, joinedPosition, selected, type, styler}) => styler.append(
+		className: ({backgroundOpacity, highlighted, joinedPosition, selected, type, styler, size}) => styler.append(
 			backgroundOpacity,
+			size,
 			type,
 			(joinedPosition && 'joined' + cap(joinedPosition)),  // If `joinedPosition` is present, prepend the word "joined" to the variable, so the classes are clearer.
-			{highlighted, selected}
+			{
+				highlighted,
+				selected
+			}
 		),
-		minWidth: ({children}) => (!children)
+		minWidth: ({children}) => (React.Children.count(children) === 0)
```

Comment:
This computation is not exactly the same.  Passing an empty string `''` will yield one child but would previously have set `minWidth`.  What was the motivation for this change?

Reply:
I'd added that. I thought it would do a better job at catching empty children.

Reply:
Is this change in function acceptable or should we revert it?

Reply:
Alternately...  Add `|| children === ''`?